src/features/Students/StudentsMetrics/index.jsx

```diff
@@ -2,28 +2,34 @@ import React, { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { Card, CardGrid, ToggleButton } from '@edx/paragon';
 import { ToggleButtonGroup } from 'react-paragon-topaz';
-import { fetchMetricsData } from 'features/Students/data/thunks';
+import { fetchClassesMetricsData, fetchStudentsMetricsData } from 'features/Students/data/thunks';
 
 import 'features/Students/StudentsMetrics/index.scss';
 
 const StudentsMetrics = () => {
   const dispatch = useDispatch();
-  const stateMetrics = useSelector((state) => state.students.metrics.data);
+  const selectedInstitution = useSelector((state) => state.main.selectedInstitution);
+  const stateStudentsMetrics = useSelector((state) => state.students.studentsMetrics.data);
+  const stateClassesMetrics = useSelector((state) => state.students.classesMetrics.data);
```

Comment:
I'm not huge fan about name this things as `state...` and the reason of this is because we're using the `useSelector` so we it is understood we're dealing with any state, maybe just `studentsMetrics` is more clear for this case, what do you think?

```suggestion
  const studentsMetrics = useSelector((state) => state.students.studentsMetrics.data);
  const classesMetrics = useSelector((state) => state.students.classesMetrics.data);
```